modules/release-notes/pages/release-notes.adoc

```diff
@@ -1,5 +1,11 @@
 = Release Notes
 
+New updates to TigerGraph Cloud, both to the user experience and the backend capability, are regularly released.
+This page provides a summary of each update upon release.
+
+Solutions created before an update will not be automatically upgraded.
```

Comment:
I feel like upgrade is a specific term that isn't necessarily related to a cloud update.

We can be a little more specific about what you mean by upgrade.

When we say upgrade I think it is usually related to upgrading from one version to another like 3.6.0 to 3.6.1. But cloud updates can touch more than just that.